code/b-parasite/src/main.c

```diff
@@ -62,8 +62,13 @@ static void rtc_callback() {
   prst_adc_soil_moisture_t soil_read = prst_adc_soil_read(batt_read.voltage);
   prst_pwm_stop();
   nrf_gpio_pin_clear(PRST_FAST_DISCH_PIN);
+
+  nrf_gpio_pin_set(PRST_PHOTO_V);                                         // set GPIO for photoresistor HIGH
+  prst_adc_photo_sensor prst_adc_photo_read(double battery_voltage);      // read PHOTO SENSOR value
+  nrf_gpio_pin_clear(PRST_PHOTO_V);                                       // not sure need this one

```

Comment:
I think it's good to have it here, yes. Maybe the pin will be automatically powered off in deep sleep, but I still think it's good to power it off as soon as we're done reading it. It's a quick win for power saving.